cmd/vela-server/main.go

```diff
@@ -157,6 +157,22 @@ func main() {
 			Name:    "vault-prefix",
 			Usage:   "vault prefix for k/v secrets. e.g. secret/data/<prefix>/<path>",
 		},
+		&cli.StringFlag{
+			EnvVars: []string{"VELA_SECRET_VAULT_AUTH_METHOD", "SECRET_VAULT_AUTH_METHOD"},
+			Name:    "vault-auth-method",
+			Usage:   "auth method to utilize to obtain token",
+		},
+		&cli.StringFlag{
+			EnvVars: []string{"VELA_SECRET_VAULT_AWS_ROLE", "SECRET_VAULT_AWS_ROLE"},
+			Name:    "vault-aws-role",
+			Usage:   "vault role to connect to the auth/aws/login endpoint with",
+		},
+		&cli.DurationFlag{
+			EnvVars: []string{"VELA_SECRET_VAULT_RENEWAL", "VELA_SECRET_VAULT_RENEWAL"},
```

Comment:
Should these be the same?

Based off your pattern above, I'm thinking it should look like:

```suggestion
			EnvVars: []string{"VELA_SECRET_VAULT_RENEWAL", "SECRET_VAULT_RENEWAL"},
```

Reply:
updated